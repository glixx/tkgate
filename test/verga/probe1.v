module top;
  integer i,a;

  initial
    begin
	$tkg$probe("bob",a,i);

	for (i = 0;i < 20;i = i + 1)
	  begin
	    if ( (i & 1) == 0)
	      a <= i;

	    #1;
	  end
    end
endmodule
