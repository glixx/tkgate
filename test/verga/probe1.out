valueof top.a 32'hx @ 0
valueof top.i 32'hx @ 0
tell bob top.i 32'h0 @ 0
tell bob top.a 32'h0 @ 0
tell bob top.i 32'h1 @ 1
tell bob top.i 32'h2 @ 2
tell bob top.a 32'h2 @ 2
tell bob top.i 32'h3 @ 3
tell bob top.i 32'h4 @ 4
tell bob top.a 32'h4 @ 4
tell bob top.i 32'h5 @ 5
tell bob top.i 32'h6 @ 6
tell bob top.a 32'h6 @ 6
tell bob top.i 32'h7 @ 7
tell bob top.i 32'h8 @ 8
tell bob top.a 32'h8 @ 8
tell bob top.i 32'h9 @ 9
tell bob top.i 32'ha @ 10
tell bob top.a 32'ha @ 10
tell bob top.i 32'hb @ 11
tell bob top.i 32'hc @ 12
tell bob top.a 32'hc @ 12
tell bob top.i 32'hd @ 13
tell bob top.i 32'he @ 14
tell bob top.a 32'he @ 14
tell bob top.i 32'hf @ 15
tell bob top.i 32'h10 @ 16
tell bob top.a 32'h10 @ 16
tell bob top.i 32'h11 @ 17
tell bob top.i 32'h12 @ 18
tell bob top.a 32'h12 @ 18
tell bob top.i 32'h13 @ 19
tell bob top.i 32'h14 @ 20
